### ISSUE TITLE
Suspicious activity detected

### ISSUE THREAD

**Issue by baclaeys** [immense/Remotely#675]
My Remotely server is hosted locally and only connects to local clients within our network.  I received an email from our cyber security department stating that two of our Remotely clients were trying to communicate with 108.177.122.127 over port 19302.  Based on my research the client is trying to get its public IP address from Google.  Assuming this is correct, is there a way to disable these communication attempts between Remotely clients with outside IP addresses.  I have no need to for this function since everything is local.

I am using a fairly old container to run Remotely.  The version is Version: 2021.08.17.1435.  I have been hesitant to update it, but if migrating to the latest version from Immense would fix this I would be willing to give it a shot.

**Comment by bitbound** [comment]
The activity is totally normal for that version of Remotely.  By default, it uses WebRTC for streaming the desktop, and WebRTC uses STUN servers for negotiation while trying to establish a connection.  19302 is the port that the Google STUN servers use.

Here's a brief summary of how STUN and TURN work:  https://www.twilio.com/docs/stun-turn/faq

I can't remember if that version had a global option for turning off WebRTC.  But if so, you could turn it off.

Otherwise, the newer versions of Remotely don't use WebRTC.  Microsoft abandoned the library I was using, and I couldn't find a suitable replacement, so I removed it.  

Edit:  19302 is the port that Google uses for all their STUN servers.  It's not the default for STUN in general.

**Comment by baclaeys** [comment]
Thank you for your response.  I was able to find the line in the configuration to disable webRTC.  As an extra precaution, I also changed the url for the IceServers to point to my Remotely server.  This did not seem to cause any problems.

As far as updating the container, it was much simpler than I thought it would be.  In my docker-compose.yml file I changed the image from "translucency/remotely:latest" to "immybot/remotely:latest".  I left the ports and volumes the same.  When I restarted the container everything came back up with the latest container.

Old:
version: "2"
services:
  remotely:
    image: translucency/remotely:latest
    ports:
      - 5000:5000
    volumes:
      - /remotely-data:/remotely-data
      - 
New:
version: "2"
services:
  remotely:
    image: immybot/remotely:latest
    ports:
      - 5000:5000
    volumes:
      - /remotely-data:/remotely-data

My concern was changing the image name.  I did not know if it would start up properly once I pulled the new image.  This may not be the case for everyone, but I hope this can help someone else.
